internal/eval/eval_test.go

```diff
@@ -473,6 +473,34 @@ func testEvalEXPIRE(t *testing.T, store *dstore.Store) {
 			input:  []string{"EXISTING_KEY", strconv.FormatInt(-1, 10)},
 			output: []byte("-ERR invalid expire time in 'expire' command\r\n"),
 		},
+		"invalid expiry time exists - empty string": {
+			setup: func() {
+				key := "EXISTING_KEY"
+				value := "mock_value"
+				obj := &object.Obj{
+					Value:          value,
+					LastAccessedAt: uint32(time.Now().Unix()),
+				}
+				store.Put(key, obj)
+
+			},
+			input:  []string{"EXISTING_KEY", ""},
+			output: []byte("-ERR value is not an integer or out of range\r\n"),
```

Comment:
This is incorrect, redis gives the following output when running EXPIRE command with empty expiry: `ERR wrong number of arguments for command`

Reply:
Verified check below ss
![image](https://github.com/user-attachments/assets/d31697ee-2de4-417b-a90e-7aa84447ce8f)
And the same mentioned in 
https://github.com/redis/redis/blob/unstable/tests/unit/expire.tcl#L240

Reply:
Ah I was treating it as missing expiry time instead of putting empty string. Thanks for clarifying.